proto/proto/penumbra/core/transaction/v1alpha1/transaction.proto

```diff
@@ -127,9 +138,9 @@ message TransactionView {
   crypto.v1alpha1.Fee fee = 4;
   // A list of clues for use with Fuzzy Message Detection.
   repeated crypto.v1alpha1.Clue fmd_clues = 5;
-  // An optional plaintext memo. It will only be populated if there are
+  // An optional view of a transaction memo. It will only be populated if there are
   // outputs in the actions of this transaction.
-  optional bytes memo = 6;
+  MemoView memo_view = 6;
```

Comment:
Should this be `optional MemoView memo_view`? I don't know if the difference is significant for a sub-message vs a scalar field, but in the `TransactionBody` the `optional` is used to signal presence or absence, which might otherwise not be accessible depending on the language-specific proto implementation.